utility/log.py

```diff
@@ -15,13 +15,17 @@
     - local file
     - logstash server
 """
+import inspect
 import logging
+import os
 from copy import deepcopy
 from typing import Any, Dict
 
 from .config import TestMetaData
 
-LOG_FORMAT = "%(asctime)s - %(levelname)s - %(name)s:%(lineno)d - %(message)s"
+LOG_FORMAT = "%(asctime)s - %(levelname)s - %(message)s"
```

Comment:
Can you add Doc string describing the **message** like what format it will be

Reply:
Updated doc string